docs/technical-design/launch-darkly-testing-approach.md

```diff
@@ -91,8 +224,8 @@ export REACT_APP_LD_CLIENT_ID='Place Launch Darkly ID here'
     })
   ```
 
-### Adding feature flags
-Adding a new feature flag for testing is automatically done once you add the flag into list of flags in the `flags.ts` constants file located in `app-web/src/common-code/featureFlags`.
+## Adding and removing feature flags
+Adding a new feature flag for testing is automatically done once you add the flag into list of flags in the `flags.ts` constants file located in `app-web/src/common-code/featureFlags`. Removing the flag from `flags.ts` can be done by removing it from the list. 
+
+**We should not be editing the `flag`s and `defaultValues` contained in `flags.ts` specifically for testing feature flags.** Because configuration of all the LaunchDarkly testing implementation heavily rely on these values, any change would cause many tests to fail. **Ideally we would only add, remove or edit them when application code relating to those flags changes.**
 
```

Comment:
@haworku Just went into some. more details on why not to change those values for testing.